components/resources/library/src/androidMain/kotlin/org/jetbrains/compose/resources/FontResources.android.kt

```diff
@@ -5,6 +5,7 @@ import androidx.compose.runtime.remember
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.text.font.*
 
+@ExperimentalResourceApi
```

Comment:
`MalformedXMLException` isn't marked with it.

Also `sealed class Resource`, but marking it Experimental can be too much, not sure 🤔.